### PR TITLE
Fixing Host Status endpoint bug (#653)

### PR DESCRIPTION
**Pull request by mathewc**
https://github.com/Azure/azure-webjobs-sdk-script/issues/653

### CODE REVIEW

File: test/WebJobs.Script.Tests/AdminControllerTests.cs [hunk 1]
```diff
@@ -16,5 +22,40 @@ public void AdminController_HasAuthorizationLevelAttribute()
             AuthorizationLevelAttribute attribute = typeof(AdminController).GetCustomAttribute<AuthorizationLevelAttribute>();
             Assert.Equal(AuthorizationLevel.Admin, attribute.Level);
         }
+
+        [Fact]
+        public async Task HostStatus_StillAccessible_WhenHostInstanceNull()
```
Comment (mathewc) [inline test/WebJobs.Script.Tests/AdminControllerTests.cs]: This test repros the bug and failed consistently before this fix

**Comment by fabiocav** [comment]
:shipit:

**Comment by brettsam** [comment]
:shipit:

**Comment by brettsam** [comment]
Did you ever commit this? I know I fixed the same thing in my commit, but is it worth it to merge this anyway so we get the new test?

**Comment by mathewc** [comment]
I couldn't get the test to stabilize with the rest of the tests so it is shelved for now. All the e2e tests are in a "house of cards" condition currently, as you can see by the failing App Veyor build. The addition of each new server based tests destabilizes the rest. We need to address this ASAP. I think we're going to have to do isolated runs of each e2e suite. Shouldn't be too hard to accomplish that.

**Pull request closed without merging**